docroot/themes/custom/dcc_theme/css/base.css

```diff
@@ -15,4 +15,9 @@
 
 .region-header .logo img {
     min-width: 40px;
+}
+
+.js-form-type-radio > label,
+.js-form-type-checkbox > label {
+    display: inline !important;
```

Comment:
Please don't use important. Make a more specific rule here.  Empty line missing at the end of the file.